python/cugraph/cugraph/dask/centrality/betweenness_centrality.py

```diff
@@ -0,0 +1,218 @@
+        of the list for estimation: the list should contain vertex
+        identifiers. If k is None (the default), all the vertices are used
+        to estimate betweenness.  Vertices obtained through sampling or
+        defined as a list will be used as sources for traversals inside the
+        algorithm.
+
+    normalized : bool, optional (default=True)
+        If True normalize the resulting betweenness centrality values
+
+    endpoints : bool, optional (default=False)
+        If true, include the endpoints in the shortest path counts.
+    
+    seed : optional (default=None)
+        if k is specified and k is an integer, use seed to initialize the
+        random number generator.
+        Using None defaults to a hash of process id, time, and hostname
+        If k is either None or list: seed parameter is ignored
+
+    Returns
+    -------
+    betweenness_centrality : dask_cudf.DataFrame
+        GPU distributed data frame containing two dask_cudf.Series of size V:
+        the vertex identifiers and the corresponding betweenness centrality values.
+
+        ddf['vertex'] : dask_cudf.Series
+            Contains the vertex identifiers
+        ddf['betweenness_centrality'] : dask_cudf.Series
+            Contains the betweenness centrality of vertices
+
+    Examples
+    --------
+    >>> import cugraph.dask as dcg
+    >>> import dask_cudf
+    >>> # ... Init a DASK Cluster
+    >>> #    see https://docs.rapids.ai/api/cugraph/stable/dask-cugraph.html
+    >>> # Download dataset from https://github.com/rapidsai/cugraph/datasets/..
+    >>> chunksize = dcg.get_chunksize(datasets_path / "karate.csv")
+    >>> ddf = dask_cudf.read_csv(datasets_path / "karate.csv",
+    ...                          chunksize=chunksize, delimiter=" ",
+    ...                          names=["src", "dst", "value"],
+    ...                          dtype=["int32", "int32", "float32"])
+    >>> dg = cugraph.Graph(directed=True)
+    >>> dg.from_dask_cudf_edgelist(ddf, source='src', destination='dst')
+    >>> pr = dcg.betweenness_centrality(dg)
+
+    """
+
+    # FIXME: Add a warning stating that weights are not supported in this
+    # this implementation
+
+    client = input_graph._client
+
+    if input_graph.store_transposed is True:
+        warning_msg = (
+            "Betweenness centrality expects the 'store_transposed' flag "
+            "to be set to 'False' for optimal performance during "
+            "the graph creation"
+        )
+        warnings.warn(warning_msg, UserWarning)
+
+    if not isinstance(k, (dask_cudf.DataFrame, dask_cudf.Series)):
+        if not isinstance(k, (cudf.DataFrame, cudf.Series)):
+            if isinstance(k, list):
+                k_dtype = input_graph.nodes().dtype
+                k = cudf.Series(k, dtype=k_dtype)
+                # convert into a dask_cudf
+                k = dask_cudf.from_cudf(k, input_graph._npartitions)
```

Comment:
Why do we create it into dask objects here only for  list objects but let cuDF objects as is which i think will fail. Can we add a test for cuDF.Series.

Reply:
Also we discussed that distributing cuDF data like this is slow not sure about the latency requirements of this algo.

Reply:
For the first concern you mentioned, you are right and that is now fixed I think.

Reply:
I added tests for different vertex list type